<?xml version="1.0" encoding="UTF-8"?>
<launch>
	
	<node pkg="diplom" type="localization.py" name="robot_localization" output="screen" >
		<param name="map_name" value="map_willowgarage_10_11" type="str" />
		<param name="gazebo_model_name" value="lidar" type="str" />
		<param name="hist_pos_limits" value="60" type="int" />
		<param name="hist_orient_limits" value="20" type="int" />
		<param name="laser_min" value="0.1" type="double" />
		<param name="laser_max" value="20.0" type="double" />
		<param name="laser_amount" value="1000" type="int" />
		<param name="step_x" value="0.5" type="double"/>
		<param name="step_y" value="0.5" type="double"/>
		<param name="threshold" value="0.90" type="double"/>
		<param name="func_compress_pos" value="absolute" type="str" />
		<param name="func_compress_orient" value="minInSector" type="str" />
		<param name="func_match_pos" value="absolute" type="str" />
		<param name="func_match_orient" value="absolute" type="str" />
	</node>
	
</launch>
